_config.yml

```diff
@@ -22,6 +22,7 @@ disqus_shortname: em-devblog
 disqus_iframe_url: "https://ericmccormick.io/disqus_comments.html"
 gist_embed_url: "https://ericmccormick.io/gist_embed.html"
 url: "https://edm00se.io" # the base hostname & protocol for your site
+enable_pwa: true
```

Comment:
I turned this into a config option to be able to turn off the feature should things go bad.